value_sensors/template.lua

```diff
@@ -9,8 +9,13 @@ function ValueSensor.new(name)
         ["name"] = name,
         ["display_name"] = { "sensor."..name..".name" },
         ["format_key"] = "sensor."..name..".format",
+        ["color"] = nil,
     }
 
+    function sensor:get_font_color()
+        return self.color
+    end
```

Comment:
Similarly to the above, there is probably not need to have an accessor function (one exists for `sensor:get_line` because that is expected to be overridden).